src/main/java/net/liquidpineapple/pang/screens/GameOverScreen.java

```diff
@@ -0,0 +1,88 @@
+package net.liquidpineapple.pang.screens;
+
+import lombok.extern.slf4j.Slf4j;
+import net.liquidpineapple.pang.Application;
+import net.liquidpineapple.pang.InputHandler;
+import net.liquidpineapple.pang.objects.NumberToken;
+
+
+import javax.imageio.ImageIO;
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * Class that represents the Controls screen.
+ * @author Jaap-Jan
+ * @date 21-9-2016.
+ */
+@Slf4j
+public class GameOverScreen extends Screen {
+    private static ArrayList<NumberToken> Places;
+    /**
+     * Constructor for the ControlsScreen.
+     */
+    public GameOverScreen() {
+        try {
+            backgroundImage = ImageIO.read(Level.class.getResource("/sprites/gameover.png"));
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        //These are from right to left.
+        Places = new ArrayList<>(9);
+        NumberToken nine = new NumberToken(524,300);
```

Comment:
Same as in WinScreen, (which i guess you helped Tim with), why not:
`Places.add( new NumberToken(492,300))` etc.
and 
```
for (NumberToken token : Places()) {
            output.objectList.add(token);
        }
```